client/app/components/routes/select-operator/form/UserOperator.tsx

```diff
@@ -25,8 +25,8 @@ async function getBusinessStructures() {
   );
 }
 
-export async function getUserOperatorFormData(id: number | undefined) {
-  if (!id) return {};
+export async function getUserOperatorFormData(id: number | string) {
+  if (!id || id === "create") return {};
```

Comment:
@BCerki here is the check we do so we don't get an error attempting to fetch `create` as an id. We do very similar for the `/[ID]` route on the `Operations` form.

Reply:
This is changed to `isNaN(Number(id))` since Sepehr's recent PR added another check for a string.